src/Http/ResponseMatcher/ExpressionResponseMatcher.php

```diff
@@ -11,7 +11,7 @@
 
 namespace FOS\HttpCacheBundle\Http\ResponseMatcher;
 
-use Sensio\Bundle\FrameworkExtraBundle\Security\ExpressionLanguage as SecurityExpressionLanguage;
+use FOS\HttpCacheBundle\Security\ExpressionLanguage as SecurityExpressionLanguage;
```

Comment:
do we still need this class or can we remove it in favor of https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/HttpFoundation/RequestMatcher/ExpressionRequestMatcher.php